crates/ruff/src/rules/flake8_todos/rules/todos.rs

```diff
@@ -358,6 +358,8 @@ fn static_errors(
             } else {
                 trimmed.text_len()
             }
+        } else if trimmed.starts_with('@') {
+            trimmed.text_len()
```

Comment:
So the `trimmed` variable, if there's a `(` but no `)` essentially treats the entire rest of the comment as the author. In the case of `# TODO (evan this is some other text`, we can't know if "evan this is some other text" is the author or not. 

We should probably treat `author_end` as the next whitespace character to restrict the author name to a word if we're using the `@` delimiter. For example, `# TODO @evan this is some other text` should treat `@evan` as the author

Reply:
That does make sense to me. 👍🏻 should I change that (and the line above my addition) or will you do it after the pull-request?

Reply:
I'd prefer changing it now so that we don't have any false positives!